terraform/README.md

```diff
@@ -374,8 +374,7 @@ below is an example in the aoc repo workflow, you can also configure it in your
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+          role-to-assume: ${{ secrets.ASSUMABLE_ROLE_ARN }}
```

Comment:
```suggestion
          role-to-assume: ${{ secrets.ASSUMABLE_ROLE_ARN }}
          role-session-name: RelevantNameForWorkflow
```

We should probably start adding session names to all of our workflows to help identify which workflow/job/step is using the role.